buildSrc/src/main/kotlin/ProjectExtensions.kt

```diff
@@ -67,7 +67,9 @@ inline fun <reified T : BaseExtension> Project.setupBase(
         // Generate smaller bytecode by not generating runtime not-null assertions.
         "-Xno-call-assertions",
         "-Xno-param-assertions",
-        "-Xno-receiver-assertions"
+        "-Xno-receiver-assertions",
+        // https://blog.jetbrains.com/kotlin/2022/02/kotlin-1-6-20-m1-released/#prototype-of-context-receivers-for-kotlin-jvm
+        "-Xcontext-receivers"
```

Comment:
https://blog.jetbrains.com/kotlin/2022/02/kotlin-1-6-20-m1-released/#prototype-of-context-receivers-for-kotlin-jvm